src/ovs/aca_vlan_manager.cpp

```diff
@@ -243,6 +259,13 @@ int ACA_Vlan_Manager::delete_l2_neighbor(string virtual_ip, string virtual_mac,
   ACA_OVS_L2_Programmer::get_instance().execute_openflow_command(
           cmd_string, culminative_time, overall_rc);
 
+  // delete arp entry in arp responder for the l2 neighbor
+  stArpCfg.mac_address = virtual_mac;
+  stArpCfg.ipv4_address = virtual_ip;
+  stArpCfg.vlan_id = internal_vlan_id;
+
+  ACA_ARP_Responder::get_instance().delete_arp_entry(&stArpCfg);
+
```

Comment:
please add the corresponding log for consistency:

"delete_l2_neighbor arp entry with ip = %s, ...